docs/consensus/overview.md

```diff
@@ -14,13 +14,70 @@ In PoAS, farmers (not miners) store as many unique pieces of the blockchain hist
 <!-- ![ConsensusPhases](../../src/Images/Consensus_Phases.png) -->
 
 Formally, PoAS is a three phase protocol, consisting of:
-- an initial **Genesis** phase (starting the chain)
 - a recurring deterministic **Archiving** phase (given new blocks of the chain, constructing a canonical history)
 - a unique setup or **Plotting** phase done individually by each farmer (given the canonical history of the blockchain, generate a unique replica (the plot) and store it on disk)
-- a probabilistic audit phase, known as **Farming** (given a challenge from a secure randomness beacon, audit the plot for a solution that satisfies some threshold, return a proof, and propose a block). The audit is based on a recurring slot challenge from a secure randomness beacon, with a frequency of one timeslot per second. 
+- a probabilistic audit phase, known as **Farming** (given a challenge from a secure randomness beacon, audit the plot for a solution that satisfies some threshold, return a proof, and propose a block). The audit is based on a recurring slot challenge from a secure randomness beacon, with a frequency of one timeslot per second.
+
+There is also an initial [**Genesis** phase](genesis.md) that starts the chain.
 
 ## *Dilithium*
 
 *Dilithium* is a second-generation PoAS consensus algorithm, which uses erasure coding and KZG commitments for distributed archiving, while combining polynomial encoding with an ASIC-resistant proof-of-space for plotting. The protocol represents a significant step forward in security and user experience for Subspace Network participants. *Dilithium* is also designed to be SSD-friendly, further enhancing energy-efficiency and decentralization. 
 
-For those familiar with our initial consensus design, *Dilithium* fulfills all the fundamental ideas described in the original [whitepaper](https://subspace.network/news/subspace-network-whitepaper) but implements them better.
\ No newline at end of file
+For those familiar with our initial consensus design, *Dilithium* fulfills all the fundamental ideas described in the original [whitepaper](https://subspace.network/news/subspace-network-whitepaper) but implements them better.
+
+
+## Technical Overview
+
+Similar to other proof-of-capacity projects, the resource that participants allocate to secure the Subspace network is
+disk space. As a resource, disk space is widely distributed and in general the notion of an ASIC does not apply to
+storage. As such, the protocol has the Subspace protocol has the potential to be highly decentralized and more fair than
```

Comment:
But ASIC does apply to the plotting function. Not sure the second "in general the notion of an ASIC does not apply to storage" is needed.

Reply:
While not explicit, this sentence softly criticises proof of stake (the initial distribution problem) and proof of work (high barrier to entry), so I'm keen to keep it.

I do think that there's value in highlighting that storage is not ASIC-able. And since this is very high level I don't feel that an explanation of the computation needed for farming is in place. So I just tried not to say something incorrect.